src/PKSim.Core/Services/CKDDiseaseStateImplementation.cs

```diff
@@ -111,6 +162,53 @@ private void updateParameter(IParameter parameter, double value)
          parameter.IsFixedValue = false;
       }
 
+      public Individual CreateBaseIndividualForPopulation(Individual originalIndividual)
+      {
+         //we need to create a new individual WITHOUT CKD and set all percentiles as in the original individuals. Other parameters, value wil be taken as is
+         var originData = originalIndividual.OriginData.Clone();
+
+         //remove the disease state to create a healthy Individual
+         originData.DiseaseState = null;
+         var healthyIndividual = _individualFactory.CreateAndOptimizeFor(originData, originalIndividual.Seed);
+
+         var allCKDParameters = parametersChangedByCKDAlgorithmAsList(healthyIndividual);
+
+         var allHealthyDistributedParameters = _containerTask.CacheAllChildrenSatisfying<IDistributedParameter>(healthyIndividual, x => !allCKDParameters.Contains(x));
```

Comment:
@Yuri05 I am not sure that we need to differentiate the percentile. This is what we discussed a few days ago but I can't see why anymore

Reply:
Because if we shift a distributed parameter in the baseIndividual, we want the same shif